spec/dummy/app/assets/config/manifest.js

```diff
@@ -0,0 +1 @@
+//
```

Comment:
Introduced since Sprockets 3.

![Screen Shot 2021-07-11 at 11 55 40 PM](https://user-images.githubusercontent.com/796573/125228854-8bec5500-e2a3-11eb-8a78-659e2dc661f9.png)